actionview/lib/action_view/template.rb

```diff
@@ -324,8 +329,10 @@ def handle_render_error(view, e) #:nodoc:
       end
 
       def locals_code #:nodoc:
+        locals = @locals.delete_if { |key| RUBY_KEYWORDS.include?(key.downcase) }
```

Comment:
@matthewd this will handle `<%= render 'template', locals: { CLASS: "food", Alias: "bar" }` which cause the same misleading syntax error.

I'd be OK changing this to not handle those edge cases though, please let me know.